gradle.properties

```diff
@@ -2,21 +2,21 @@ org.gradle.caching=true
 puppycrawlCheckstyleVersion=8.18
 group=org.ballerinalang
 version=1.0.8-SNAPSHOT
-ballerinaLangVersion=2.0.0-alpha5-SNAPSHOT
-stdlibIoVersion=0.6.0-alpha4-SNAPSHOT
-stdlibOsVersion=0.8.0-alpha4-SNAPSHOT
+ballerinaLangVersion=2.0.0-alpha6-SNAPSHOT
+stdlibIoVersion=0.6.0-alpha5-SNAPSHOT
+stdlibOsVersion=0.8.0-alpha5-SNAPSHOT
 stdlibUuidVersion=0.10.0-alpha4-SNAPSHOT
```

Comment:
@aashikam shouldn't these be updated?

Reply:
Yes. Could you please update the missed ones :)